src/App.js

```diff
@@ -31,10 +33,36 @@ class App extends React.Component {
     })
   }
 
-  render(){
+  // handleNumberHorns = event => {
+  //   let newNumsArr = totalImageArray;
+  //   let selected = event.target.horns;
+
+  //   this.setState({
+  //     selected: event.target.horns
+  //   })
+  // }
+
```

Comment:
- Great job on the lab so far!

- We can remove this commented out code and instead, most likely use it inside `Main.js`. We will want to do our filtering of horns there 👍